pyluna-common/luna/common/utils.py

```diff
@@ -329,3 +332,16 @@ def cli_runner(cli_kwargs: dict, cli_params: List[tuple], cli_function: Callable
         # json.dump(kwargs, fp, indent=4, sort_keys=True)
     
     logger.info("Done.")
+
+
+def load_func(dotpath : str):
```

Comment:
Yeah, so there are a couple levels here. So, this `load_func` needs to be used since yaml interprets things like `nn.Module` as a string (by default, I think it’s possible to write a custom parser that could deal with this directly). However, if you want to also pass parameters to the loaded function in the yaml, that’s not possible with this function. So, for example, I wanted to pass something like`tune.choice([16, 32])` to the batch size parameter, `tune.choice` must be passed  separately  from `[16, 32]`. So that would be an additional feature that might be possible to add in the future.

Reply:
For tune.choice like things, just pass the expected arguments (e.g. [16,32]), and detect when the user wants to only run for a singular value